editor/src/com/talosvfx/talos/editor/addons/scene/widgets/directoryview/DirectoryViewWidget.java

```diff
@@ -423,7 +424,57 @@ public void drop (DragAndDrop.Source source, DragAndDrop.Payload payload, float
 
 					openDirectory(targetItem.fileHandle.path());
 				} else {
-					logger.info("TODO COPY INTO DIRECTORY and do asset stuff");
+					if (payloadObject instanceof GlobalDragAndDrop.ArrayDragAndDropPayload) {
+						Array<GlobalDragAndDrop.BaseDragAndDropPayload> items = ((GlobalDragAndDrop.ArrayDragAndDropPayload) payloadObject).getItems();
```

Comment:
Payload might be a single file, so this will miss that case

Reply:
Fixed!